test/scripts/e2e_subs/rest-participation-key.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# TIMEOUT=300
+
+my_dir="$(dirname "$0")"
+#"$my_dir/rest.sh" "$@"
+source "$my_dir/rest.sh" "$@"
+
+date "+$0 start %Y%m%d_%H%M%S"
+
+# Use admin token for both get and post
+export USE_ADMIN=true
+
+pushd "${TEMPDIR}" || exit
+
+FIRST_ROUND=0
+# A really large (but arbitrary) last valid round
+LAST_ROUND=1200000
+
+NAME_OF_TEMP_PARTKEY="tmp.${FIRST_ROUND}.${LAST_ROUND}.partkey"
+
+algokey part generate --first ${FIRST_ROUND} --last ${LAST_ROUND} --keyfile ${NAME_OF_TEMP_PARTKEY} --parent ${ACCOUNT}
+
+popd || exit
+
+call_and_verify "Get List of Keys" "/v2/participation" 200 'Address'
+
+call_post_and_verify "Install a basic participation key" "/v2/participation" 200 ${NAME_OF_TEMP_PARTKEY} 'partId'
+
+call_and_verify "Get List of Keys" "/v2/participation" 200 'Address'
```

Comment:
If you can't get the ID from `call_and_verify`, maybe change the check here for something that wasn't available in the empty response?